src/terraform-provider-signalform/signalform/time_chart.go

```diff
@@ -116,28 +155,27 @@ func timeChartResource() *schema.Resource {
 				Type:     schema.TypeSet,
 				Optional: true,
 				Elem: &schema.Resource{
+					SchemaVersion: 1,
+					MigrateState:  resourceAxisMigrateState,
 					Schema: map[string]*schema.Schema{
 						"min_value": &schema.Schema{
 							Type:        schema.TypeFloat,
 							Optional:    true,
-							Default:     -math.MaxFloat32,
-							Description: "The minimum value for the right axis",
+							Description: "The minimum value for the left axis",
```

Comment:
nitpick: these are on `axis_right`, but description says "left axis". same with the two below it